api/envoy/config/filter/network/http_connection_manager/v2/http_connection_manager.proto

```diff
@@ -484,6 +484,10 @@ message HttpConnectionManager {
   // with `prefix` match set to `/dir`. Defaults to `false`. Note that slash merging is not part of
   // `HTTP spec <https://tools.ietf.org/html/rfc3986>` and is provided for convenience.
   bool merge_slashes = 33;
+
+  // The Name of the extension to use for request ID utilities. This includes operations such as
+  // generation, validation, and associated tracing operations.
+  string request_id_utils_name = 36;
```

Comment:
We should use a typed_config here (with any Any param) to allow for future configuration. This is how are we standardizing on loading extensions now. Please see other examples.